src/coordinator/tsdb/remote/compressed_codecs_test.go

```diff
@@ -189,7 +89,7 @@ func validateSeries(t *testing.T, it encoding.SeriesIterator) {
 	defer it.Close()
 
 	expectedValues := [60]float64{}
-	for i := 0; i < 30; i++ {
+	for i := 2; i < 30; i++ {
```

Comment:
Is this because of the timing thing with the block start vs series iterator start?

Reply:
Yeah, the first two entries in the block are outside the iterator range